akka-stream/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -159,3 +159,10 @@ ProblemFilters.exclude[Problem]("akka.stream.StreamRefMessages*")
 
 # #27266 changes to streams internals
 ProblemFilters.exclude[Problem]("akka.stream.impl.*")
+
+# added private[this] field to public class, shouldn't have more impact than a potential naming clash
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.stage.OutHandler.akka$stream$stage$OutHandler$$_lastCancellationCause")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.stage.OutHandler.akka$stream$stage$OutHandler$$_lastCancellationCause_=")
```

Comment:
This should no longer be necessary (#27529)

Reply:
Thx, I'll rebase to latest master and fix this

Reply:
@raboof fixed - should be ready to be merged :)